python/ml_wrappers/model/image_model_wrapper.py

```diff
@@ -316,8 +316,11 @@ def _get_device(device: str) -> str:
        or type(device) == int
        or device is None):
         if device == Device.AUTO.value:
-            return (Device.CUDA.value if torch.cuda.is_available()
-                    else Device.CPU.value)
+            return Device.CPU.value
```

Comment:
I think we want to keep the old logic but maybe not pass auto instead but cpu directly from wherever this is called?